src/resources/views/crud/columns/table_relationships.blade.php

```diff
@@ -0,0 +1,41 @@
+@php
+      $value = data_get($entry, $column['name']);
+      $columns = $column['columns'];
+@endphp
+
+<span>
+
+    @if ($value && count($columns))
+
+        @includeWhen(!empty($column['wrapper']), 'crud::columns.inc.wrapper_start')
+
+        <table class="table table-bordered table-condensed table-responsive-sm table-sm table-striped m-b-0">
+		<thead>
+			<tr>
+				@foreach($columns as $tableColumnKey => $tableColumnDefinition)
+                    <th>{{ is_array($tableColumnDefinition) ? $tableColumnDefinition['label'] : $tableColumnDefinition }}</th>
+                @endforeach
+			</tr>
+		</thead>
+		<tbody>
+			@foreach ($value as $tableRow)
+                <tr>
+				@foreach($columns as $tableColumnKey => $tableColumnDefinition)
+                    <td>
+                        @if (isset($tableColumnDefinition['callback']) && is_callable($tableColumnDefinition['callback']))
+                            {{ $tableColumnDefinition['callback'](Arr::get($tableRow, $tableColumnDefinition['attribute'] ?? $tableColumnKey), $tableRow) }}
```

Comment:
Ok so this is great - I think it's a really great minimal way to output the contents of a column on the related item. But. Someone, someday, will definitely ask this 😆 so I'm going to do it ahead of time. **Couldn't we use the existing column types here?**

So similar to the repeatable _field_ type, this table _column_ type could take as columns a _column definition array_, right? And then here, in the actual blade file, instead of doing this if/else with the callback or not, it could just `@include()` the column type the user specified (fallback to text). Right?

I'm not saying it's a good idea at the moment. I'm just thinking out loud, that if the `repeatable` field type works that way, perhaps the users will think this is supposed to work in a similar way too. Right?

What you think @mgralikowski ?
How about you @pxpm - what do you think?

Reply:
Indeed, I am pretty sure that this only makes sense if the same columns we already support are the ones you are going to provide in table.
That way we could easily access `wrapper` in columns and create the links etc. 

The way I see it is something like:

```php
CRUD::column('schoolRoles')
->type('relationship_table')
->columns(
    CRUD::column('id'), 
    CRUD::column('name'),
    CRUD::column('school.name')->type('relationship')->wrapper([
                       'href' => function ($crud, $column, $entry, $related_key) {
                        return backpack_url('school/'.$related_key.'/edit');
                    }])
) ;
```

Reply:
I tried with CRUD::column object but I can't access column's attributes because it is protected. There is no getter method so I am not able to do sth like this:
`{{ $column->attributes['label] }}` or `{{ $column->label }}`.